src/hyperlink/_url.py

```diff
@@ -815,9 +815,18 @@ class URL(object):
           that starts with a slash.
        userinfo (Text): The username or colon-separated
           username:password pair.
-       uses_netloc (bool): Indicates whether two slashes appear
-          between the scheme and the host (``http://eg.com`` vs
-          ``mailto:e@g.com``). Set automatically based on scheme.
+       uses_netloc (Optional[bool]): Indicates whether ``://`` (the "netloc
+           separator") will appear to separate the scheme from the *path* in
```

Comment:
Because `:` is always present when there's a scheme, wouldn't `//` be the "netloc separator"?

Reply:
I think I'd still go with `://` here, because while you're right about the colon, `//` can appear lots of other places in the URL without being a netloc separator, but an unescaped `://` always means "netloc separator"

Reply:
Sure, if you feel like that's clearer.